tfjs-backend-webgpu/src/index.ts

```diff
@@ -47,7 +47,9 @@ if (device_util.isBrowser() && isWebGPUSupported()) {
     } else {
       console.warn(
           `This device doesn't support timestamp-query extension. ` +
-          `Zero will shown for the kernel time when profiling mode is` +
+          `Start Chrome browser with flag ` +
+          `--disable-dawn-features=disallow_unsafe_apis then try again. ` +
+          `Or zero will shown for the kernel time when profiling mode is` +
           `enabled. Using performance.now is not workable for webgpu since` +
           `it doesn't support synchronously to read data from GPU.`);
```

Comment:
comment unrelated to this change: I don't see why async means you can't use performance.now. You can get a timestamp before issuing the work, await the work being completed, and get another timestamp.

Reply:
@kainino0x It's limited by the tfjs implementation. You can find the detailed reason here https://github.com/tensorflow/tfjs/pull/4862#issuecomment-809003392 and https://github.com/tensorflow/tfjs/pull/4862#issuecomment-809004617

Reply:
Thanks for the background, I think I understand the issue is that apps using the tf.js api are calling many ops synchronously so you can't just `await` on `onSubmittedWorkDone` between ops like you could with dataSync.

Reply:
Yes, we can't use `await` in a sync function `profileKernel`. And if we put it in the async `time`, we can't make sure the correct time is collected as comment https://github.com/tensorflow/tfjs/pull/4862#issuecomment-809004617 said.